src/bundle/Resources/public/js/scripts/admin.content.edit.js

```diff
@@ -179,6 +182,11 @@
         btn.addEventListener('click', clickHandler, false);
     });
 
-    doc.querySelector('.ibexa-content-edit-content').addEventListener('scroll', fitHeader, false);
+    const contentEditContent = doc.querySelector('.ibexa-content-edit-content');
+
+    if (contentEditContent) {
```

Comment:
But here you can use optional chaining!

(prophet of optional chaining mode on)